packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1827,7 +1838,11 @@ class TextInput {
         (_currentConnection!._client as DeltaTextInputClient).updateEditingValueWithDeltas(deltas);
         break;
       case 'TextInputClient.performAction':
-        _currentConnection!._client.performAction(_toTextInputAction(args[1] as String));
+        if (args[1] as String == 'TextInputAction.commitContent') {
+          _currentConnection!._client.commitContent(args[2] as Map<String, dynamic>);
```

Comment:
What if the returned content here doesn't match the contentCommitMimeTypes?  I understand that the engine should prevent that from happening, but maybe it's worth an assert somewhere.

Reply:
Added